packages/perspective-viewer-d3fc/src/js/legend/legend.js

```diff
@@ -65,3 +66,9 @@ export function filterData(settings, data) {
     }
     return useData;
 }
+
+export function configureHidden(oldSettings, newSettings) {
+    const areSplitsEqual = oldSettings && oldSettings.splitValues && _.isEqual(oldSettings.splitValues, newSettings.splitValues);

```

Comment:
Underscore is a big thing to add to the package for this one little thing, and ES2015 does away with most of its benefits. It should be pretty easy to compare 2 simple arrays without it.